blocks/log.py

```diff
@@ -245,3 +254,18 @@ def __iter__(self):
 
     def get_status(self):
         return self._status
+
+    def to_dataframe(self):
+        return DataFrame.from_dict(self._storage, orient='index')
+
+
+def to_dataframe(log):
+    """Convert a log into a :class:`.DataFrame`."""
```

Comment:
You'll need to add `pandas` to InterSphinx by adding `'pandas': 'http://pandas.pydata.org/pandas-docs/stable/'` to the `intersphinx_mapping` dictionary in `docs/conf.py`. After that, you can link to the pandas documentation by using 
```
:class:`~pandas.DataFrame`
```